Makefile

```diff
@@ -123,6 +123,7 @@ test: all
 	$(MAKE) cctest
 	$(PYTHON) tools/test.py --mode=release -J \
 		addons doctool inspector known_issues message pseudo-tty parallel sequential
+	$(MAKE) doclint
```

Comment:
Maybe rename to `mdlint`? I assume we want to lint every md, including non-doc files.